docs/api/cases/cases-api-find-cases.asciidoc

```diff
@@ -0,0 +1,171 @@
+[[cases-api-find-cases]]
+== Find cases API
+++++
+<titleabbrev>Find cases</titleabbrev>
+++++
+
+Retrieves a paginated subset of cases. By default, the first page is returned
+with 20 results per page.
+
+=== Request
+
+`GET <kibana host>:<port>/api/cases/_find`
+
+=== Prerequisite
+
+You must have `read` privileges for the *Cases* feature in the *Management*,
```

Comment:
Each privilege corresponds to one owner. For example, if you want to get only all the oy11 cases you need to put the owner as `observability`. With the owner set as `observability`, you need to have `read` privileges for Observability Cases. The same goes for security solution and stack management. I am not sure if this sentence captures that.

Reply:
Good point, I've updated in 1f36fd9da172bf319fa6acd1b31a11cf46624d9a